### ISSUE TITLE
NVDA cursor reading doesn't work in the latest version, version 0.1.2

### ISSUE THREAD

**Issue by MyNameIsTrez** [Crimso777/Factorio-Access#19]
I've tried creating a new world, restarting NVDA, using NVDA's insert plus two hotkey to turn "speak type characters" off, and none of those made me able to read with the cursor. I redownloaded the latest version of Factorio, version 1.1.61, along with version 0.1.2 of Factorio Access, and NVDA still doesn't work for me.

**Comment by Crimso777** [comment]
Fixed and pushed.
